blog/2023-03-10-conda-survey.md

```diff
@@ -0,0 +1,194 @@
+
+**What do we do with all this?**
+
+Let's start by noting that the Python community strongly prioritizes interoperability between Python packaging tool, but that support for multiple languages is at the bottom.  This is good news and bad news for conda.  Conda already works well with Pip.  The downside is that most PSF respondents don't prioritize support for multiple languages, which is one of conda's superpowers.
+
+PSF respondents would also clearly prefer to have a single packaging toolkit.  This is not necessarily bad news for conda, as the conda ecosystem addresses concerns that Pip does not.  The PSF survey does not ask "What toolkit would you prefer that to be." However, given the Python community's current usage of Pip/Venv our guess is that they would say "Pip/Venv", and that is a challenge we need to address.  We hope the current much wider us of Pip/Venv is merely a byproduct of the wide range of Python packages available via Pip and that this is a Python community survey, rather than a widespread preference for the capabilities of Pip/Venv over conda.  But, from these surveys we can't tell that.
+
+# When is conda used?
+
+## Industries and Ambits
+
+[![Top industries](/img/2023-03-10-conda-survey/industries-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1700458323)
+
+Just over 40% of respondents are connected to *academia*. Since the student plus professor/instructor pool adds up to less than 25%, a significant slice of these academics come from other job functions. Specific domains make up the long tail of responses.
+
+When asked in [what settings participants use conda](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=720256469) 81% said *at work*, while only 18% said *school-related.*  72% use conda on *open-source* projects.  53% use if for "personal* projects.
+
+## Domains
+
+The survey used the [categories from the packages directory on the Anaconda Nucleus site](https://anaconda.cloud/package-categories) to ask in what application areas conda is used in
+
+[![What types of applications do you use conda in?](/img/2023-03-10-conda-survey/what-types-of-applications-do-you-use-conda-in-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1170448145)
+
+Data Usage & ETL (67%, Extract, Transform and Load) and Machine Learning & AI (63%) were the most common applications.  Visualization (54%), Packaging (54%) and Scientific (53%) make up the next largest groups.
+
+# What is conda used with?
+
+## Packages
+
+The conda survey asked people to list their top 5 packages.
+
+[![Top 5 packages](/img/2023-03-10-conda-survey/top-5-packages.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=968125862)
```

Comment:
This image 404s on the preview.

Reply:
I have no idea what is happening here.  Can you recheck?

Reply:
Weird, it works now! But that's good!